docs/package.json

```
@@ -15,7 +15,8 @@
         "build:sass": "sass --style=compressed src/assets/scss:src/assets/css --no-source-map",
         "build:eleventy": "npx @11ty/eleventy",
         "start": "npm-run-all build:sass --parallel watch:*",
-        "build": "npm-run-all build:sass build:eleventy images"
+        "build": "npm-run-all build:sass build:eleventy images",
+        "lint:scss": "stylelint \"**/*.scss\""
```

```suggestion
        "lint:scss": "stylelint \"**/*.scss\"",
        "lint:fix:scss": "npm run lint:scss -- --fix"
```

Copy & paste from https://github.com/eslint/eslint/pull/16379#discussion_r987281984 as GitHub now considers it outdated and doesn't allow to apply the change.